.github/workflows/ci.yml

```diff
@@ -0,0 +1,54 @@
+name: CI
+
+on:
+  push:
+   branches:
+     - "master"
+     - "develop"
+  pull_request:
+      types: [opened, synchronize, reopened]
+jobs:
+  build:
+    defaults:
+      run:
+        shell: bash
+        working-directory: "/opt/carma/"
+    runs-on: ubuntu-latest
+    container:
+      image: usdotfhwastoldev/carma-base:develop
+      env:
+        INIT_ENV: "/home/carma/.base-image/init-env.sh"
+        ROS_2_ENV: "/opt/ros/foxy/setup.bash"
+        TERM: xterm
+      options: "--user root"
+    steps:
+    - name: Checkout ${{ github.event.repository.name }}
+      uses: actions/checkout@v3.3.0
+      with:
+        path: src/${{ github.event.repository.name }}
+        fetch-depth: 0
+    - name: Move source code
+      run: mv $GITHUB_WORKSPACE/src /opt/carma/
+    - name: Checkout dependencies
+      run: |
+        source "$INIT_ENV"
+        source "$ROS_2_ENV" 
+        ./src/${{ github.event.repository.name }}/carma-messenger-core/docker/checkout.bash -r /opt/carma/
+    - name: Build 
+      run: |
+        source "$INIT_ENV"
+        source "$ROS_2_ENV" 
+        sed -i '/colcon build/ s/$/ --packages-up-to carma-messenger cpp_message emergency_response_vehicle_plugin j2735_convertor asn1c traffic_incident truck_inspection_plugin/' /home/carma/.ci-image/engineering_tools/code_coverage/make_with_coverage.bash

```

Comment:
This can be removed, make_with_coverage will try to build all the packages in directory

Reply:
we are still executing make_with_coverage.bash as sed -i command is used to add a list of package names to lines in make_with_coverage.bash that contain the text "colcon build. we are modifying script during build time without effecting the file and include additional packages when building with colcon build.

Reply:
So the point was that we dont need to be manually listing all the packages. Is there a reason we need to do that?